resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -135,10 +135,10 @@
 options.dist = "#{options.operating_system}#{options.os_version}" if options.dist.nil?
 
 if options.debug
-  STDERR.puts "=========================="
-  STDERR.puts "OPTIONS HASH"
-  STDERR.puts options
-  STDERR.puts "=========================="
+  puts "=========================="
```

Comment:
Why change debug output from stderr to stdout?

Reply:
I originally had it in stderr because I was having issue with puts statements not making it to stdout but that seems to be working. Is stderr better for debug output? I always forget what's standard

Reply:
Probably fine as long as it requires enabling the debug output

Reply:
Yeah, debug is disabled by default